src/dz13/ValueCalculator.java

```diff
@@ -0,0 +1,39 @@
+package dz13;
+
+
+
+public class ValueCalculator  {
+    float[] massiv;
+    int size = 1000000;
+    int half = 500000;
+    float[] arr1 = new float[half];
+    float[] arr2 = new float[half];
+    long time;
+
+    public void method() {
+        time = System.currentTimeMillis();
+
+        massiv = new float[size];
+        for (int i=0;i<size;i++){
+            massiv[i] = 1;
+        }
+
+        System.arraycopy(massiv, 0, arr1, 0, half);
+        System.arraycopy(massiv, half, arr2, 0, half);
+    }
+    public void method1() {
```

Comment:
Imagine a situation when you should work with the code of another dev
What is `method1`. You may broke brain for understanding what is it